packages/admin-next/dashboard/src/v2-routes/invite/invite.tsx

```diff
@@ -0,0 +1,393 @@
+import { zodResolver } from "@hookform/resolvers/zod"
+import { UserRoles } from "@medusajs/medusa"
+import { Alert, Button, Heading, Input, Text } from "@medusajs/ui"
+import { AnimatePresence, motion } from "framer-motion"
+import { Trans, useTranslation } from "react-i18next"
+import { Link, useSearchParams } from "react-router-dom"
+import * as z from "zod"
+
+import { useState } from "react"
+import { useForm } from "react-hook-form"
+import { decodeToken } from "react-jwt"
+import { Form } from "../../components/common/form"
+import { LogoBox } from "../../components/common/logo-box"
+import { isAxiosError } from "../../lib/is-axios-error"
+import { useV2AcceptInvite, useV2CreateAuthUser } from "../../lib/api-v2"
+
+const CreateAccountSchema = z
+  .object({
+    email: z.string().email(),
+    first_name: z.string().min(1),
+    last_name: z.string().min(1),
+    password: z.string().min(1),
+    repeat_password: z.string().min(1),
+  })
+  .superRefine(({ password, repeat_password }, ctx) => {
+    if (password !== repeat_password) {
+      ctx.addIssue({
+        code: z.ZodIssueCode.custom,
+        message: "Passwords do not match",
+        path: ["repeat_password"],
+      })
+    }
+  })
+
+type DecodedInvite = {
+  id: string
+  jti: UserRoles
+  exp: string
+  iat: number
+}
+
+export const Invite = () => {
+  const [searchParams] = useSearchParams()
+  const [success, setSuccess] = useState(false)
+
+  const token = searchParams.get("token")
+  const invite: DecodedInvite | null = token ? decodeToken(token) : null
+  const isValidInvite = invite && validateDecodedInvite(invite)
+
+  return (
+    <div className="min-h-dvh w-dvw bg-ui-bg-base relative flex items-center justify-center p-4">
+      <div className="flex w-full max-w-[300px] flex-col items-center">
+        <LogoBox
+          className="mb-4"
+          checked={success}
+          containerTransition={{
```

Comment:
+1 think that we could have something as `<Fade direction= "up" />` or at least easing functions in the UI lib

Reply:
Do we want to address such things in UI for v2, or should we wait till after? 

And if so, do we know where we want to place these?

Reply:
I would define these in `medusajs/ui` but probably not necessary for v2 atm

Reply:
If the animation is ever re-used I think it would make sense to move it to a file in `@medusajs/dashboard`, but it's a one off animation, so don't see a reason to move it into a separate file atm. Feel free to move it to a const outside of the JSX for better readability though.

And I would prefer to not move any framer logic into `@medusajs/ui`. We don't use Framer in the project, only a small number of CSS animations. If people want to add animations on top of that they can then use framer or their preferred animation component, but I don't think it should be something that is built-in, or force people to use a specific animation library ✌️

Reply:
@kasperkristensen it doesn't necessarily need to be part of `medusajs/ui`, it can be a separate `animations` folder in the admin, but my point is that it would be nice to see what is extractable. And definitely not urgent for v2, just a general comment after looking at the code.